cmd/api/database.go

```diff
@@ -88,16 +88,18 @@ func searchById(id uuid.UUID) (Book, error) {
 }
 
 /* Returns filtered content of database in a list of books*/
-func listBooks(name string, minPrice32, maxPrice32 float32) ([]Book, error) {
+func listBooks(name string, minPrice32, maxPrice32 float32, sortBy, sortDirection string) ([]Book, error) {
 	if name == "" {
 		name = "%"
 	}
+	name = fmt.Sprint("%", name, "%")
```

Comment:
If `name` is empty you will end up with this:
```go
name = fmt.Sprint("%", "%", "%")
```
Does this works as expected? seems a little wrong to have triple `%` though... you could change it to the following and avoid that case.

```suggestion
	if name != "" {
		name = fmt.Sprint("%", name, "%")
	} else {
	        name = "%"
        }

Reply:
It works, but your way is better.